src/compiler/commandLineParser.ts

```diff
@@ -150,6 +150,14 @@ namespace ts {
             category: Diagnostics.Command_line_Options,
             description: Diagnostics.Whether_to_keep_outdated_console_output_in_watch_mode_instead_of_clearing_the_screen,
         },
+        {

```

Comment:
There is already incremental included in this list why is there another entry?

Reply:
guess that was automerged and I missed it - pushed a fix for it

Reply:
Can you please existing entry instead? Is there reason this needs to move ?

Reply:
better now?